### PR TITLE
Add Learn PWA: Part 20, Tools and Debug.

### LINKED ISSUE #6499: Learn PWA: 21. Tools and debug

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #6499 

Changes proposed in this pull request:

-
-
-

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by jpmedley** [comment]
@rachelandrew Please don't edit this yet.

**Comment by jpmedley** [comment]
@rachelandrew I forgot to add you to this one. 21 and 22 are approved, but I don't want to merge those until this one is done.

File: src/site/content/en/learn/pwa/tools-and-debug/index.md [hunk 1]
```diff
@@ -1,11 +1,253 @@
...
+
+### Installation debugging
+
+On Android devices using [WebAPK installation mode](/learn/pwa/installation/#webapks), you can access a list of installed apps by browsing to `about:webapk` on Chromium browsers.
+
+You will see the current update status and request a WebAPK update from here.
+
+On desktop computers, you can see a user-friendly list of installed PWAs by browsing to `about:apps`, and a debug version by browsing to `about:app-service-internals`.
+
+{% Img src="image/RK2djpBgopg9kzCyJbUSjhEGmnw1/exwzkevoGROml8uH5ZxL.png", alt="Google Chrome WebAPK debug screen on Android", width="800", height="400" %}
```
Comment (rachelandrew) [inline src/site/content/en/learn/pwa/tools-and-debug/index.md:199]:
```suggestion
{% Img src="image/RK2djpBgopg9kzCyJbUSjhEGmnw1/exwzkevoGROml8uH5ZxL.png", alt="Google Chrome WebAPK debug screen on Android.", width="800", height="400" %}
```

File: src/site/content/en/learn/pwa/tools-and-debug/index.md [hunk 2]
```diff
@@ -1,11 +1,253 @@
...
+
+### Service worker tools
+
+What you can do on Safari (both on macOS and remotely for iOS and iPadOS) is to open an inspector window for a service worker that is currently running.
+
+From Safari on macOS, the `Develop`, `Service Workers` menu will list all the currently running service worker sessions. You have to select the device to inspect in the `Develop` menu for remote inspection. Service workers' contexts will appear in the same submenu as window contexts under the origin's or PWA's installed name.
+
+If you select one of them, Safari will open a new window with a restricted inspector, including only `Consoles`, `Sources`, and `Network Tabs`.
+
+{% Img src="image/RK2djpBgopg9kzCyJbUSjhEGmnw1/jho0nKOzPq9XPH8g24Hk.png", alt="A Service Worker Web Inspector from a PWA in Safari", width="800", height="427" %}
```
Comment (rachelandrew) [inline src/site/content/en/learn/pwa/tools-and-debug/index.md:220]:
```suggestion
{% Img src="image/RK2djpBgopg9kzCyJbUSjhEGmnw1/jho0nKOzPq9XPH8g24Hk.png", alt="A Service Worker Web Inspector from a PWA in Safari.", width="800", height="427" %}
```

**Review by rachelandrew — APPROVED** [review]

**Comment by rachelandrew** [comment]
@jpmedley this is good to go (ignore the approvals missing check)

**Pull request merged**